components/ListThumbnail.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core'
+import React from 'react'
+import {
+  POST_THUMBNAIL_HEIGHT,
+  POST_THUMBNAIL_BORDER_RADIUS,
+} from '../lib/constants'
+import { POST_THUMBNAIL } from '../lib/utils'
```

Comment:
That uppercase style should be used only for constants. I'll change it, that's okay.